cdatweb/apps/vtk_view/views.py

```diff
@@ -61,7 +61,7 @@ def vtk_test(request, test="cone"):
 def vtkweb_launcher(request):
     """Proxy requests to the configured launcher service."""
     import requests
-    VISUALIZATION_LAUNCHER = 'http://aims1.llnl.gov/vtk'
+    #VISUALIZATION_LAUNCHER = 'http://aims1.llnl.gov/vtk'
```

Comment:
@jbeezley I tried local_settings but it didn't work me (may be I didn't do it right or something)